admin/routes/test-developer.js

```diff
@@ -166,12 +36,12 @@ router.get('/choose-check-window', isAuthenticated(rolesConfig.ROLE_TEST_DEVELOP
       formIds.push(req.query['check-form'])
     } else {
       console.error(new Error('No formIds selected'))
-      return res.redirect('manage-check-forms')
+      return res.redirect('upload-and-view-forms')
     }
   }
 
   try {
-    forms = await CheckForm.getActiveForms({_id: formIds}).sort({_id: 1}).exec()
+    forms = await checkFormService.getActiveForms({_id: formIds}).sort({_id: 1}).exec()
```

Comment:
The checkFormService should be returning JSON objects, but the line below makes me think otherwise...